jwql/website/manage.py

```diff
@@ -47,11 +47,12 @@
         'outputs': 'outputs',
         'preview_image_filesystem': 'preview_images',
         'thumbnail_filesystem': 'thumbnails'
-        }
+    }
 
     for directory in ['filesystem', 'outputs', 'preview_image_filesystem', 'thumbnail_filesystem']:
         symlink_location = os.path.join(os.path.dirname(__file__), 'apps', 'jwql', 'static', directory_mapping[directory])
         if not os.path.exists(symlink_location):
+            print("{} does not exist".format(symlink_location))
```

Comment:
This is fine, it looks like it still attempts to create a symlink. If the `symlink_location` doesn't exist, this print statement will alert the user, but does `os.symlink` not throw an error?

Reply:
So, this was a modification that I made to the auto-generated file to find out what it was failing on, and why, because I couldn't figure out why my local JWQL (with config file) couldn't run the django manage script, and it turns out that if you create a symlink, but it's an incorrect symlink (as in it points to something that doesn't exist), then os.path.exists() will return False on the symlink, even though the symlink is there in the filesystem, and this helped me diagnose what the issue actually was. I didn't want to actually change the way the file behaved, because again it's auto-generated by django so I wasn't comfortable fiddling with it too much.

Basically, the issue that I had was that:
- If you have directories (e.g.) internal/ and actual/
- If you create actual/data/
- If you create the symlink internal/data/ but you point it at actual/Data

Then calling os.path.exists() on internal/data will return False, but trying to create the symlink internal/data pointing to actual/data will fail because internal/data already exists. I'm not actually sure why the script wasn't working (why it didn't create things correctly the first time, that is) but I was able to solve it by making sure all the symlinks were set up and valid before running, and then I forgot to remove that diagnostic line.

Reply:
Cool, thanks for the explanation. Let's just make sure to remove this line before merging, thank you!